content/200-concepts/100-components/02-prisma-client/037-relation-queries.mdx

```diff
@@ -324,7 +324,7 @@ const user = await prisma.user.findFirst({
 
 In [3.0.1](https://github.com/prisma/prisma/releases/3.0.1) and later, you can [`include` or `select` a count of relations](aggregation-grouping-summarizing#count-relations) alongside fields - for example, a user's post count.
 
-<CodeWithResult>
+<CodeWithResult outputResultText="query">
```

Comment:
I think you forgot line 257 @jharrell

Reply:
I think that one actually makes sense! It's an error rather than a result of the query. Maybe "Show error result"?

Reply:
Updated!